README.md

```diff
@@ -16,16 +16,17 @@ You can access the detailed rule description directly from your editor, using th
 
 Out of the box, SonarLint automatically checks your code against the following rules:
 
-- [JavaScript rules](https://rules.sonarsource.com/javascript)
-- [TypeScript rules](https://rules.sonarsource.com/typescript)
 - [C rules](https://rules.sonarsource.com/c)
 - [C++ rules](https://rules.sonarsource.com/cpp)
 - [CSS rules](https://rules.sonarsource.com/css)
-- [Python rules](https://rules.sonarsource.com/python)
-- [Java rules](https://rules.sonarsource.com/java)
+- [Go rules](https://rules.sonarsource.com/go)
 - [HTML rules](https://rules.sonarsource.com/html)
```

Comment:
Add item for Go and sort existing list alphabetically

Reply:
Thank you for the alphabetic sorting 🥇